process-engine/process_engine_core
# ✨ Get instances and tasks by identity

**Changes:**

1. Use actual return types for the `startProcessInstance` use cases in the `ExecuteProcessService`
2. Fix the basic `start` UseCase on the `ExecuteProcessService`
    - The function now resolves immediately after starting the ProcessInstance
    - The functions return value now contains all relevant infos about that ProcessInstances, wrapped in a `ProcessStartedMessage` object
3. Refactor `EventAggregatorSettings`:
    - Merge `routePaths` and `messagePaths` into `messagePaths`
    - Rename `routeParams` to `messageParams`
4. Add `flowNodeInstanceId` and `identity` to all `BpmnMessages`
5. Add `identity` to all `SystemMessages`
6. Add `identity` and `parentProcessInstanceId` to the `FlowNodeInstance` type
7. Simplify some business rules in the `ResumeProcessService`, made possible by the new Properties on the `FlowNodeInstance` data model.
8. Fix the `onError` state transition for `CallActivities`, `Subprocesses` and internal `ServiceTasks`.

**Issues:**

Part of https://github.com/process-engine/process_engine_runtime/issues/198

PR: #227

## How can others test the changes?

Check the return value for `executeProcessService.start()`.
All other changes are internal and should not be externally visible.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
